test/unit/imgix-provider.test.jsx

```diff
@@ -0,0 +1,109 @@
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+import ReactImgix from "../../src/index"
+import { ImgixProvider } from "../../es/HOCs/imgixProvider"
+
+const providerProps = {
+  domain: "sdk-test.imgix.net",
+  sizes: "100vw",
+}
+
+const imageProps = {
+  src: "https://assets.imgix.net/examples/pione.jpg",
+  sizes: "50vw",
+}
+
+describe('ImgixProvider', () => {
+
+  test('should not have context value defined if Provider has no props', () => {
```

Comment:
So while these tests work, I think these tests are brittle because we're depending on the order of composition of the higher order components on `ReactImgix`. For example, I think if we add another higher order component or change the order around, these tests would break. A potential better solution would be a) traverse the tree to check props only on a specific component name, or b) use some sort of spy (I know we had issues with this). 

There's no change needed because it's too much effort to fix for the benefit, but something to keep in mind for next time round.